### PR TITLE
MACT-4: Implement Account Contacts step

### PR DESCRIPTION
**Pull request by guillegr123**

### CODE REVIEW

File: submodules/wizard/wizard.js [hunk 1]
```diff
@@ -287,24 +288,136 @@ define(function(require) {
...
+		 * Utility funcion to validate Account Contacts form and extract data
+		 * @param  {jQuery} $template  Step template
+		 * @param  {Object} args  Wizard's arguments
+		 * @param  {Object} args.data  Wizard's data that is shared across steps
+		 * @returns  {Object}  Object that contains the updated step data, and if it is valid
+		 */
+		wizardAccountContactsUtil: function($template, args) {
+			var self = this,
+				$form = $template.find('form'),
+				validateForm = monster.ui.validate($template.find('form')),
```
Comment (azefiel) [inline submodules/wizard/wizard.js]:
```suggestion
				validateForm = monster.ui.validate($form),
```

File: submodules/wizard/views/step-accountContacts.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	<div class="step-title">
+		{{i18n.accountsApp.wizard.steps.accountContacts.title}}
+	</div>
+	<div class="step-body">
+		<form id="form_account_contacts" name="form_account_contacts">
+			<div class="step-section">
+			{{#with i18n.accountsApp.wizard.steps.accountContacts as |accountContactsI18n|}}
+				{{#with @root.i18n.accountsApp.wizard.steps.general as |generalI18n|}}
+				<div class="section-title">
+					{{accountContactsI18n.salesRep.title}}
```
Comment (azefiel) [inline submodules/wizard/views/step-accountContacts.html]: I believe the spec calls for the sales rep section to not be exposed for now.

File: submodules/wizard/views/step-accountContacts.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+				</div>
+				<div class="row-fluid">
+					<div class="span4">
```
Comment (azefiel) [inline submodules/wizard/views/step-accountContacts.html]: Flexbox + percentage width + media queries could have been used for a more custom layout here, think about it next time.

File: i18n/en-US.json [hunk 4]
```diff
@@ -355,7 +355,25 @@
 				},
 				"accountContacts": {
 					"label": "Account Contacts",
-					"description": "Note important contacts for this new account."
+					"description": "Note important contacts for this new account.",
```
[the remark this answers is not in the document]
Comment (guillegr123) [reply i18n/en-US.json]: From the mockups :sweat_smile:

File: submodules/wizard/views/step-accountContacts.html [hunk 5]
```diff
@@ -0,0 +1,58 @@
+{{#*inline "inputPartial"}}
+	<div class="span4">
+		<label for="{{name}}">{{coalesce i18n.label name}}</label>
```
[the remark this answers is not in the document]
Comment (guillegr123) [reply submodules/wizard/views/step-accountContacts.html]: OK. Should this rule be applied for `name` attributes too?

[the remark this answers is not in the document]
Comment (azefiel) [reply submodules/wizard/views/step-accountContacts.html]: Nope, `name` values are usually either in snake case or camel case depending on where it comes from/goes to.

File: submodules/wizard/views/step-accountContacts.html (hunk 2 above)
Comment (guillegr123) [reply submodules/wizard/views/step-accountContacts.html]: I think it says not to save that info yet.

> At the time of writing, this field should not be saved as we are unsure when to display the parent section.

Should I remove the section for the time being? If so, do I remove the related logic too?

File: submodules/wizard/views/step-accountContacts.html (hunk 3 above)
Comment (guillegr123) [reply submodules/wizard/views/step-accountContacts.html]: I thought about it, but then remembered that Bootstrap was already available, and opted for it to not to increase the CSS styles. I will use custom styles for the next steps, and retrofit this and the previous one.

File: submodules/wizard/views/step-accountContacts.html (hunk 2 above)
Comment (azefiel) [reply submodules/wizard/views/step-accountContacts.html]: Just hide it for now, this way you shouldn't have to touch the validation logic.

Comment (guillegr123) [reply submodules/wizard/views/step-accountContacts.html]: Done :+1: I added an `if` block in the template to not render the sales representative section for the time being.

**Pull request merged**